scripts/dataset/publish/index.js

```diff
@@ -1,36 +1,14 @@
-import fsApi from 'fs';
-import path from 'path';
-import url from 'url';
+import publishGitHub from './github/index.js';
+import publishGitLab from './gitlab/index.js';
 
-import config from 'config';
-import { Octokit } from 'octokit';
+export default function publishRelease({ archivePath, releaseDate, stats }) {
+  if (process.env.OTA_ENGINE_GITHUB_TOKEN) {
+    return publishGitHub({ archivePath, releaseDate, stats });
```

Comment:
I would find it important to document either through a message here or in the README that `OTA_ENGINE_GITHUB_TOKEN` takes precedence over `OTA_ENGINE_GITLAB_TOKEN` if both are defined 🙂